src/lib/functions/runtimes/js/builders/netlify-lambda.ts

```diff
@@ -14,19 +14,21 @@ export const detectNetlifyLambda = async function ({ packageJson } = {}) {
     return false
   }
 
+  const program = new Command()
```

Comment:
using this constructor is also cleaner since it avoids mutating the exported singleton